Configuration/Generator/python/ZpMM_6000_14TeV_TuneCP5_cfi.py

```diff
@@ -0,0 +1,51 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         #pythiaHepMCVerbosity = cms.untracked.bool(False),
```

Comment:
Isn't false the default? Why is the purpose to uncomment this line? Did it cause problems? Same above.

Reply:
I have no idea since it was uncommented in [the original configuration](https://github.com/cms-sw/cmssw/blob/master/Configuration/Generator/python/ZpMM_2250_13TeV_TuneCUETP8M1_cfi.py#L6). I just didn't want to touch anything irrelevant to the requested update.

Reply:
If you are going to update the name of the fragment and correct the ttbar line, I would propose to turn it on to make it explicit.